README.md

```diff
@@ -110,30 +88,22 @@ Runs the docker image exposing the specified local port.
 
 1. Install prerequisites:
 
-   * Node v8 (preferably via [nvm](https://github.com/creationix/nvm))
-   * `grunt`
+   * [Hugo](https://gohugo.io/)
    * Docker
 
 2. Clone repo
     ```sh
     git clone https://github.com/decred/dcrweb
     cd dcrweb
-    yarn install
     ```
 
 3. Start development web server:
     ```sh
-    yarn dev
-    ```
-
-4. Start file watcher:
-
-    ```sh
-    grunt watch
+    bin/watch.sh
     ```
 
-You should now be able to access the site at `http://localhost:8080`
+You should now be able to access the site at `http://localhost:1313`
 
 ## License
 
-decredweb is licensed under the liberal ISC License.
+decredweb is licensed under the liberal ISC License.
```

Comment:
Not related to the work you are doing @peterzen, but while you are working on the README you might as well fix this

```suggestion
dcrweb is licensed under the liberal ISC License.
```